darts/timeseries.py

```diff
@@ -669,6 +669,39 @@ def shift(self, n: int) -> 'TimeSeries':
         new_series.index = new_time_index
         return TimeSeries(new_series, self.freq_str())
 
+    def diff(self, n: Optional[int] = 1,
+                periods: Optional[int] = 1,
+                dropna: Optional[bool] = True) -> 'TimeSeries':
+        """
+        Returns a differenced time series with the goal to make a time series stationary.
+
+        Parameters
+        ----------
+        n
+            Optionally, a signed integer indicating the number of differencing steps.
+        periods
+            Optionally, periods to shift for calculating difference.
+        dropna
+            Optionally, a boolean value indicating whether to drop the missing values created by
+            the pandas.DataFrame.diff method.
+
+        Returns
+        -------
+        TimeSeries
+            A TimeSeries constructed after differencing.
+        """
+        if not isinstance(n, int) or n < 1:
+             raise_log(ValueError("'n' must be a positive integer >= 1."))
+        if not isinstance(periods, int):
+             raise_log(ValueError("'periods' must be an integer."))
+
+        temp_df = self.pd_dataframe(copy=True)
```

Comment:
Not sure you need the copying if `diff()` already returns a copy and doesn't work in-place (to be confirmed but I guess that's the case).

Reply:
Done